### ISSUE TITLE
bug - ArrowKeysReact.config isn't define

### ISSUE THREAD

**Issue by wellDan28** [aboveyunhai/chakra-dayzed-datepicker#21]
## details: 

error raise on console when clicking on a RangeDatepicker

### build:
- vite (es-build), production
### deps: 
- react: 18.1
- @chakra-ui/react: 2.1.2
- chakra-dayzed-datepicker: ^0.2.2 - we had this issue in past versions as well

### error on console:
```
TypeError: IU.config is not a function
    at uie (chakra-dayzed-datepicker.esm.js:165:18)
    at Y8 (react-dom.production.min.js:166:137)
    at AZ (react-dom.production.min.js:289:386)
    at $Z (react-dom.production.min.js:279:389)
    at q0e (react-dom.production.min.js:279:320)
    at wC (react-dom.production.min.js:279:180)
    at Vk (react-dom.production.min.js:270:88)
    at mF (react-dom.production.min.js:272:300)
    at sp (react-dom.production.min.js:127:105)
    at react-dom.production.min.js:266:269
```

possible solution, my coworker @hexelon  solve it in this [fork](https://github.com/hexelon/chakra-dayzed-datepicker)
(here is the specific [commit](https://github.com/hexelon/chakra-dayzed-datepicker/commit/4f44e565e3975f613b54304d1fdaeb97dd7dfa15))

I can open a pr if needed, please let me know 🙏

**Comment by aboveyunhai** [comment]
A minimal repo/codesandbox will be helpful to find out the core issue. 
The error code looks like your production build is failing to include dependencies.

**Comment by wellDan28** [comment]
yeah, you're totally right, I'll should have add an example, I'll try to find time to do it, 
don't have an eta.

**Comment by dreammaker7** [comment]
@wellDan28 
Hi wellDan, 
I also encountered the problem you described, how can I avoid this error in my project until the problem is solved?

**Comment by aboveyunhai** [comment]
@dreammaker7  I assume the problem might be the dependency package is failed to import because of the bundling tools.
Since the package is relatively simple, I can just remove it and copy/paste the code from that lib into a single file, and then give the full credit. 
Then you can test it out to see if it works.

**Comment by aboveyunhai** [comment]
The bug should be resolved by v0.2.3
